oremda/event_loops/mpi/root_event_loop.py

```diff
@@ -13,9 +13,12 @@ class MPIRootEventLoop(MPIEventLoop):
     async def loop(self, rank):
         while True:
             # Wait until the node indicates it is ready
+            print(f"Waiting for {rank=} to be ready...")
             msg = await self.mpi_recv(rank)
+            print(f"Received message from {rank=}!")
             ready_msg = MPINodeReadyMessage(**msg.dict())
             queue = ready_msg.queue
+            print(f"{rank=} is ready! queue is {queue}. Waiting for input...")
```

Comment:
I left some debug output in here because it's helpful when we develop it. Feel free to let me know if you want me to remove it.